samples/ClientSample/Tcp/TcpConnection.cs

```diff
@@ -4,13 +4,15 @@
 using System.IO.Pipelines;
 using System.Net;
 using System.Net.Sockets;
+using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Connections;
+using Microsoft.AspNetCore.Connections.Features;
 using Microsoft.AspNetCore.Http.Features;
 
 namespace ClientSample
 {
-    public class TcpConnection : ConnectionContext
+    public class TcpConnection : ConnectionContext, IConnectionInherentKeepAliveFeature
```

Comment:
Wat? TCP doesn't have inherent keep-alive...

Reply:
> Add IConnectionInherentKeepAliveFeature to the tcp connection impl since Kestrel doesn't implement the heartbeat (yet).

Read my description brah

Reply:
Put it in a comment then brah